resources/views/components/accordion/mobile.blade.php

```diff
@@ -0,0 +1,25 @@
+{{--
+This is a responsive version of the accordion component.
+
+Responsive behavior adjustments:
+    - The `label` slot applies `md:cursor-auto`,
+      making it non-clickable on larger screens,
+
+    - The `content` slot uses `md:grid-rows-[1fr]`
+      to collapse the content on smaller screens,
+      overriding the default `grid-rows-[0fr]`.
+
+These changes ensure that the accordion functions only as an accordion on smaller screens.
+
+TODO: implement attribute merging on label and slot
```

Comment:
What's the follow-up on this?

Reply:
Internal reference: JFE-857

Reply:
TODO is not ready so I think have to wait when it is fixed before we can merge.